spec/policies/content_policy_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ContentPolicy, type: :policy do
+  let!(:admin) { create(:user, role: "admin") }
+  let!(:learner) { create(:user, role: 'learner') }
```

Comment:
can we follow single quoted strings for all?

Reply:
Bhaiya, I replaced all of double quoted strings.
Thank you.